account_statement_import_camt/tests/test_import_bank_statement.py

```diff
@@ -20,29 +61,34 @@ def setUpClass(cls):
         super().setUpClass()
         cls.parser = cls.env["account.statement.import.camt.parser"]
 
-    def _do_parse_test(self, inputfile, goldenfile):
-        testfile = get_module_resource(
-            "account_statement_import_camt", "test_files", inputfile
+    def _do_parse_test(self, inputfile, goldenfile, max_diff_count=None):
```

Comment:
if your intent is to still use the class you should move this to a mixin or common setup class otherwise your inheriting class will run the same tests